tests/utils.bal

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerinax/java.jdbc;
+import ballerina/sql;
+
+string url = "jdbc:sqlserver://localhost:1433;";
+string user = "sa";
+string password = "Test123#";
+
+function getMssqlClient(string? database = ()) returns jdbc:Client|error {
+    jdbc:Client dbClient;
+    if database is () {
+        dbClient = check new (url, user, password);
+    } else {
+        dbClient = check new (url + "databaseName=" + database, user, password);
+    }
+    return dbClient;
+}
+
+function queryMssqlClient(string|sql:ParameterizedQuery sqlQuery, typedesc<record {}>? resultType = (), string? database = ())
```

Comment:
Are these functions used?

Reply:
No they are not. All util functions removed in cf71558